test/deploy/kubernetes/provisioner-pods/deployment_server_node1.yaml

```diff
@@ -0,0 +1,98 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: server-node1
+  labels:
+    app: server-node1
+    id: ddd120a9e051d103c164f605615c32a4
+spec:
+  hostname: server-node1
+  restartPolicy: Never
+  nodeSelector:
+    node-name: node1
+  volumes:
+  - name: node-id
+    downwardAPI:
+      items:
+      - path: "id"
+        fieldRef:
+          fieldPath: metadata.labels['id']
+  - name: solution-config
+    configMap:
+      name: solution-config
+  - name: cortx-secret
+    secret:
+      secretName: cortx-secret
+  - name: shared-node1
+    persistentVolumeClaim:
+      claimName: shared-node1
+  - name: local-node1
+    persistentVolumeClaim:
+      claimName: local-node1
+  - name: sdb
+    persistentVolumeClaim:
```

Comment:
I doubt if server nodes (pods) need all the devices? Thats only applicable for storage and data nodes (pods)

Reply:
done